sources/parsers/inc/Evaluator.h

```diff
@@ -100,13 +100,13 @@ namespace PARSER
             /// the expected lexer interface.
 
             /// The decimal(vector) lexer.
-            class DecimalScanner *m_pDecimalLexer;
+            class DecimalScanner *m_pDecimalLexer = nullptr;
```

Comment:
Why is `class` keyword here?

Reply:
There are no includes. These are forward definitions.